uwblueprint/feeding-canadian-kids
# Auth APIs

## Notion ticket link

[Auth APIs](https://app.zenhub.com/workspaces/fck-627480c10d8974001872b0ef/issues/uwblueprint/feeding-canadian-kids/13)



## Implementation description
* Converted boilerplate code for authentication APIs (login, register, refresh, etc.) from REST APIs to Graphene



## Steps to test
1. Register a new user on local server - you should receive a verification link in the email you sign up with (make sure to check your junk mail!)
2. Login as the new user (you don't need to be verified to do this) - you should see a new dashboard
3. Refresh the page - you should stay logged in
4. Logout of your account
* None of these steps should throw any errors, UNLESS you try registering with an email that already exists (in which case you will get a popup saying that registration failed)



## What should reviewers focus on?
* Mostly any edge cases for authentication that you think our code might not address, or ways that the functionality could potentially get abused


## Checklist
